frontend/public/components/dashboard/inventory-card/inventory-item.tsx

```diff
@@ -53,9 +53,14 @@ export const InventoryItem: React.FC<InventoryItemProps> = React.memo(
     } else {
       status = children;
     }
+    let label = <>{isLoading || error ? title : `${count} ${title}`}</>;
+
+    if (link) {
+      label = <Link to={link}>{label}</Link>;
```

Comment:
The link capability is fairly limiting here, although it probably covers current requirements. `<Link>`'s `to` prop is not limited to just string.

Reply:
Yeah for this i think it's fine